source/element-kinds.md

```diff
@@ -293,7 +328,9 @@ Element parameter groups associated with this element kind are:
 (s:kicker)=
 ## Kicker Element
 
-Particle kicker element.
+A kicker element is an element that can deflect a beam in both planes. 
+It uses the hkick and vkick parameters to deflect the beam in
+horizontal and vertical directions.
 
 Under Construction...
```

Comment:
Note:  There is currently no separate parameter group (e.g. `KickerP`) for describing kicker-specific parameters.  I suppose one could re-use the bend parameter group (`BendP`), but that doesn't provide an option to give a simple _integrated_ kick.  If you like, I could draft something in `parameters/kicker.md`.

Reply:
@DavidSagan I overlooked that zero-order (dipole) multipole components are allowed in PALS.  In that case, re-using those here seems optimal.